e2e/cli-e2e/tests/help.spec.ts

```diff
@@ -1,13 +1,15 @@
 import { executeProcess } from '@code-pushup/utils';
 
 describe('CLI help', () => {
-  it('should print help', async () => {
+  it('should print help whit argument ---help', async () => {
     const { code, stdout, stderr } = await executeProcess({
       command: 'npx',
       args: ['./dist/packages/cli', '--help'],
     });
     expect(code).toBe(0);
     expect(stderr).toBe('');
-    expect(stdout).toMatchSnapshot();
+    expect(stdout).toMatchSnapshot('help');
```

Comment:
Why? You didn't update the snapshots BTW. But I don't see we shouldn't keep the default snapshot name.

![image](https://github.com/flowup/quality-metrics-cli/assets/34691111/3d1f469a-9dc7-4360-9ea0-0a7b03fb1ee3)